AISKU/Tests/es6-module-type-check/src/main.ts

```diff
@@ -0,0 +1,2 @@
+import { ContextTagKeys } from "@microsoft/applicationinsights-common";
```

Comment:
Simple example provide by the community which is consuming the generated type(s) and identifying that there was an issue with the type constraints